src/components/TrackNewExercise.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, FormEvent } from 'react'
+
+export function TrackNewExercise({
+  trackId,
+}: {
+  trackId: TrackIdentifier
+}): JSX.Element {
+  const [exampleFilename, setExampleFilename] = useState('')
+
+  const handleSubmit = (evt: FormEvent<HTMLFormElement>) => {
+    evt.preventDefault()
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
```

Comment:
Before you go and try to implement all the form state ping me, and I'll give you a few options that you can use;